gulpfile.js

```diff
@@ -10,26 +10,31 @@
 
 'use strict';
 
-const compilerPackage = require('google-closure-compiler');
 const gulp = require('gulp');
 const sourcemaps = require('gulp-sourcemaps');
-
-const closureCompiler = compilerPackage.gulp();
+const closure = require('google-closure-compiler').gulp();
+const closureOptions = {
+  compilation_level: 'ADVANCED',
+  warning_level: 'VERBOSE',
+  language_in: 'ES6_STRICT',
+  language_out: 'ES5_STRICT',
+  externs: ['externs/html-imports.js'],
+  js_output_file: 'html-imports.min.js',
+  assume_function_wrapper: true,
+  new_type_inf: true,
+  rewrite_polyfills: false,
+  new_type_inf: true,
+  module_resolution: 'NODE',
```

Comment:
This line is unnecessary I think? DIdn't see any imports in the main file

Reply:
The copy-paste curse :x will fix

Reply:
Done in https://github.com/webcomponents/html-imports/pull/75/commits/b14476ce0dc5dd4b1d93469d7abf79cd1f5a4f01, PTAL